src/compiler/utilities.ts

```diff
@@ -742,6 +742,11 @@ namespace ts {
         return false;
     }
 
+    export function isMinusPrefixUnaryExpression(node: Node): boolean {

```

Comment:
i would rather this would be isPrefixUniraryExpresssion that is a type guard, i.e. 

```ts
function isPrefixUniraryExpresssion (node: Node): node is PrefixUnaryExpression {
    return node.kind === SyntaxKind.PrefixUnaryExpression;
}
```

And used as 
```ts
 const withMinus = isPrefixUniraryExpresssion(node.parent) && node.parent.operator === SyntaxKind.MinusToken;
```